src/main.js

```diff
@@ -15,12 +14,11 @@ __webpack_nonce__ = btoa(OC.requestToken) // eslint-disable-line
 __webpack_public_path__ = generateFilePath('tables', '', 'js/')
 
 Vue.mixin({ methods: { t, n } })
-Vue.use(Vuex)
+Vue.use(createPinia())
 Vue.use(VuePapaParse)
 
 export default new Vue({
 	el: '#content',
 	router,
```

Comment:
```suggestion

const pinia = createPinia()

export default new Vue({
	el: '#content',
	router,
	pinia,
```


Plus I think if this is vue 2 you need `Vue.use(PiniaVuePlugin)`